pkgs/development/libraries/libraspberrypi/default.nix

```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
   patches = [
     (fetchpatch {
       # https://github.com/raspberrypi/userland/pull/670
-      url = "https://github.com/raspberrypi/userland/pull/670/commits/37cb44f314ab1209fe2a0a2449ef78893b1e5f62.patch";
+      url = "https://github.com/raspberrypi/userland/commit/37cb44f314ab1209fe2a0a2449ef78893b1e5f62.patch";
       sha256 = "1fbrbkpc4cc010ji8z4ll63g17n6jl67kdy62m74bhlxn72gg9rw";
```

Comment:
Did you verify that the hash is still correct?

Reply:
Yes, when I change the last character of the hash to a bogus value then you can see the real hash of the new URL, which is the exactly the same as before:

```
$ nix-build '<nixpkgs>' -A libraspberrypi.patches
these derivations will be built:
  /nix/store/v9l4rav1gf9ga2ixar3fi86zpkykn0wr-37cb44f314ab1209fe2a0a2449ef78893b1e5f62.patch.drv
these paths will be fetched (0.00 MiB download, 0.21 MiB unpacked):
  /nix/store/lis141php3j1fjmn2wdvcfq97v1nds0l-patchutils-0.3.3
copying path '/nix/store/lis141php3j1fjmn2wdvcfq97v1nds0l-patchutils-0.3.3' from 'http://homeserv:3000'...
building '/nix/store/v9l4rav1gf9ga2ixar3fi86zpkykn0wr-37cb44f314ab1209fe2a0a2449ef78893b1e5f62.patch.drv' on 'ssh://root@homeserv'...

trying https://github.com/raspberrypi/userland/commit/37cb44f314ab1209fe2a0a2449ef78893b1e5f62.patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3272  100  3272    0     0  10554      0 --:--:-- --:--:-- --:--:-- 10554
hash mismatch in fixed-output derivation '/nix/store/h9vz52l1bsh3qrv2p8188yqsc1gsfik0-37cb44f314ab1209fe2a0a2449ef78893b1e5f62.patch':
  wanted: sha256:1fbrbkpc4cc010ji8z4ll63g17n6jl67kdy62m74bhlxn72gg9r0
  got:    sha256:1fbrbkpc4cc010ji8z4ll63g17n6jl67kdy62m74bhlxn72gg9rw
error: build of '/nix/store/v9l4rav1gf9ga2ixar3fi86zpkykn0wr-37cb44f314ab1209fe2a0a2449ef78893b1e5f62.patch.drv' on 'ssh://root@homeserv' failed: hash mismatch in fixed-output derivation '/nix/store/h9vz52l1bsh3qrv2p8188yqsc1gsfik0-37cb44f314ab1209fe2a0a2449ef78893b1e5f62.patch':
  wanted: sha256:1fbrbkpc4cc010ji8z4ll63g17n6jl67kdy62m74bhlxn72gg9r0
  got:    sha256:1fbrbkpc4cc010ji8z4ll63g17n6jl67kdy62m74bhlxn72gg9rw
builder for '/nix/store/v9l4rav1gf9ga2ixar3fi86zpkykn0wr-37cb44f314ab1209fe2a0a2449ef78893b1e5f62.patch.drv' failed with exit code 1
error: build of '/nix/store/v9l4rav1gf9ga2ixar3fi86zpkykn0wr-37cb44f314ab1209fe2a0a2449ef78893b1e5f62.patch.drv' failed
```